test/data/mock-state.json

```diff
@@ -383,20 +383,30 @@
       "unconnectedAccount": true
     },
     "featureFlags": {},
-    "networkConfigurations": {
-      "testNetworkConfigurationId": {
-        "rpcUrl": "https://testrpc.com",
+    "networkConfigurationsByChainId": {
+      "0x1": {
         "chainId": "0x1",
-        "nickname": "Custom Mainnet RPC",
-        "type": "rpc",
-        "id": "testNetworkConfigurationId"
+        "name": "Custom Mainnet RPC",
+        "nativeCurrency": "ETH",
+        "defaultRpcEndpointIndex": 0,
+        "rpcEndpoints": [
+          {
+            "url": "https://testrpc.com",
+            "networkClientId": "testNetworkConfigurationId"
+          }
```

Comment:
does this need `type`?

Reply:
type is a required property and would always be defined in the real state.  I'll try to add one to this mock.